[{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/comments/56344973","html_url":"https://github.com/elastic/elasticsearch/issues/7800#issuecomment-56344973","issue_url":"https://api.github.com/repos/elastic/elasticsearch/issues/7800","id":56344973,"node_id":"MDEyOklzc3VlQ29tbWVudDU2MzQ0OTcz","user":{"login":"MikeZuehlke","id":4973326,"node_id":"MDQ6VXNlcjQ5NzMzMjY=","avatar_url":"https://avatars3.githubusercontent.com/u/4973326?v=4","gravatar_id":"","url":"https://api.github.com/users/MikeZuehlke","html_url":"https://github.com/MikeZuehlke","followers_url":"https://api.github.com/users/MikeZuehlke/followers","following_url":"https://api.github.com/users/MikeZuehlke/following{/other_user}","gists_url":"https://api.github.com/users/MikeZuehlke/gists{/gist_id}","starred_url":"https://api.github.com/users/MikeZuehlke/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/MikeZuehlke/subscriptions","organizations_url":"https://api.github.com/users/MikeZuehlke/orgs","repos_url":"https://api.github.com/users/MikeZuehlke/repos","events_url":"https://api.github.com/users/MikeZuehlke/events{/privacy}","received_events_url":"https://api.github.com/users/MikeZuehlke/received_events","type":"User","site_admin":false},"created_at":"2014-09-22T08:45:45Z","updated_at":"2014-09-22T08:52:57Z","author_association":"NONE","body":"After playing with version 1.2.4 I faced the same error. Deleting the data directory helped to resolve the issue. I tried again the versions 1.3.\\* after deleting the data directory and IT WORKS.\n","performed_via_github_app":null},{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/comments/57177704","html_url":"https://github.com/elastic/elasticsearch/issues/7800#issuecomment-57177704","issue_url":"https://api.github.com/repos/elastic/elasticsearch/issues/7800","id":57177704,"node_id":"MDEyOklzc3VlQ29tbWVudDU3MTc3NzA0","user":{"login":"clintongormley","id":56599,"node_id":"MDQ6VXNlcjU2NTk5","avatar_url":"https://avatars0.githubusercontent.com/u/56599?v=4","gravatar_id":"","url":"https://api.github.com/users/clintongormley","html_url":"https://github.com/clintongormley","followers_url":"https://api.github.com/users/clintongormley/followers","following_url":"https://api.github.com/users/clintongormley/following{/other_user}","gists_url":"https://api.github.com/users/clintongormley/gists{/gist_id}","starred_url":"https://api.github.com/users/clintongormley/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/clintongormley/subscriptions","organizations_url":"https://api.github.com/users/clintongormley/orgs","repos_url":"https://api.github.com/users/clintongormley/repos","events_url":"https://api.github.com/users/clintongormley/events{/privacy}","received_events_url":"https://api.github.com/users/clintongormley/received_events","type":"User","site_admin":false},"created_at":"2014-09-29T15:24:07Z","updated_at":"2014-09-29T15:24:07Z","author_association":"CONTRIBUTOR","body":"OK - so looks like this was caused by an invalid file name. Did you move the data directory from a linux box?  How did that directory come to exist?\n\nI'm going to close this in favour of #6736\n","performed_via_github_app":null}]